static/js/shared/ga_events.ts

```diff
@@ -67,3 +67,45 @@ export const GA_VALUE_TOOL_CHART_OPTION_SHOW_DENSITY = "show density";
 export const GA_VALUE_TOOL_CHART_OPTION_EDIT_SOURCES = "edit sources";
 export const GA_VALUE_TOOL_CHART_OPTION_FILTER_BY_POPULATION =
   "filter by population";
+
+/**
+ * GA custom events documentation
+ *
+ * Place explorer:
+ * 1. Event name: place_category_click
```

Comment:
If documenting this in the code, I would put the description for each event above that event's constant. So for this description, I would put it above line 32

Reply:
Done.